lib/oli_web/live/delivery/student/learn_live.ex

```diff
@@ -2397,7 +2397,7 @@ defmodule OliWeb.Delivery.Student.LearnLive do
     >
       <div class="dark:text-white text-[13px] font-semibold">
         <%= parse_module_total_pages(@page_metrics.total_pages_count) <>
-          if parsed_minutes, do: " · #{parsed_minutes}" %>
+          ((@page_metrics.total_duration_minutes > 0 && " · #{parsed_minutes}") || "") %>
```

Comment:
This line could be simplify in this way
```
<%= parse_module_total_pages(@page_metrics.total_pages_count) <> "#{parsed_minutes}" %>
```

if we change how `parse_card_badge_minutes` behaves in this way:

```
  defp parse_card_badge_minutes(minutes, resource_type) when is_integer(minutes) do
    clock_duration =
      minutes
      |> Timex.Duration.from_minutes()
      |> Timex.Duration.to_clock()

    case {clock_duration, resource_type} do
      {{hours, minutes, _seconds, _milliseconds}, :module} when hours > 0 ->
        " · #{hours}h #{minutes}m"

      {{_, minutes, _seconds, _milliseconds}, :module} when minutes > 0 ->
        " · #{minutes}m"

      {{hours, minutes, _seconds, _milliseconds}, :page} when hours > 0 ->
        " · #{hours}h #{minutes}m"

      {{_, minutes, _seconds, _milliseconds}, :page} when minutes > 0 ->
        " · #{minutes} min"

      {{_, _, _seconds, _milliseconds}, _} ->
        ""
    end
  end
```

Also the only difference between the resource_type is in the `" · #{minutes} min"` for `: page ` and the `" · #{minutes}m"` for `: module` if posible this could be just simply be

```
defp parse_card_badge_minutes(minutes) when is_integer(minutes) do
    clock_duration =
      minutes
      |> Timex.Duration.from_minutes()
      |> Timex.Duration.to_clock()

    case {clock_duration} do
      {{hours, minutes, _seconds, _milliseconds}} when hours > 0 ->
        " · #{hours}h #{minutes}m"

      {{_, minutes, _seconds, _milliseconds}} when minutes > 0 ->
        " · #{minutes} min"

      {{_, _, _seconds, _milliseconds}, _} ->
        ""
    end
  end
```

Either way, the `case` could be simplify.

The whole purpose of this comment is to improve the way we read the following line:


```
<%= parse_module_total_pages(@page_metrics.total_pages_count) <> "#{parsed_minutes}" %>
```

so we don't add business logic in the middle.

Reply:
I agree that we can simplify this.

I think the difference between the legend shown if the resource type is :page(**“ - #{minutes**} min”) or :module(**“ - #{minutes}m”**) is there for a reason. I don't know if unifying everything in a legend like the one you suggest (“ - #{minutes} min”) would break the design.